src/sliding-sync.ts

```diff
@@ -82,10 +82,17 @@ export interface MSC3575SlidingSyncRequest {
     clientTimeout?: number;
 }
 
+export interface MSC4186Hero {
+    user_id: string;
+    displayname: string;
+    avatar_url: string;
+}
```

Comment:
documentation please. How does this differ from the other `Hero`?

Reply:
Done in 7028a88b2

Reply:
Done in 7028a88b2

Reply:
Done in 7028a88b2